lib/rules/no-actions-hash.js

```diff
@@ -0,0 +1,60 @@
+const ember = require('../utils/ember');
+
+const ERROR_MESSAGE = 'Use the @action decorator instead of declaring an actions hash';
+
+module.exports = {
+  ERROR_MESSAGE,
+  meta: {
+    docs: {
+      description: 'Disallows the actions hash in components, controllers and routes',
+      category: 'Ember Object',
```

Comment:
@patocallaghan do you think we should use the `Ember Octane` category like the other new rules?